src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-Footer/Components/index.tsx

```diff
@@ -109,11 +109,7 @@ const PureFooter: React.FunctionComponent<Props> = ({
           </li>
           <li>
             {' '}
-            <a
-              target="_blank"
-              rel="noopener noreferrer"
-              href="https://medium.com/swlh/ignitus-connecting-students-with-opportunities-61f08e19861c"
-            >
+            <a target="_blank" rel="noopener noreferrer" href="/careerPage">
```

Comment:
Why have you added a career page link to Blog? nevertheless and the link should be `careersPage` not `careerPage`.

Reply:
I will tc of that.